README.md

```diff
@@ -1,47 +1,39 @@
-Demozoo
-=======
-
-developer installation
-----------------------
+# Demozoo
 
+## Developer installation
 The recommended way to set up a developer instance is through [Vagrant](https://www.vagrantup.com/). After installing Vagrant and VirtualBox, install as follows:
 
     git clone https://github.com/demozoo/demozoo.git
     cd demozoo
     vagrant up
 
-This will download an Ubuntu Focal64 image, install dependencies, and fetch the latest public export of the Demozoo database. To start up the site:
+This will download an Ubuntu Jelly64 image, install dependencies, and fetch the latest public export of the Demozoo database. To start up the site:
```

Comment:
Jammy, not Jelly :-)
```suggestion
This will download an Ubuntu Jammy64 image, install dependencies, and fetch the latest public export of the Demozoo database. To start up the site:
```

Reply:
D'oh. Fixed.